setup/MonsterConfig.m

```diff
@@ -50,7 +51,7 @@
 
 			% Parameters related to simulation run time
 			Runtime = struct();
-			numRounds = 10;
```

Comment:
@MonsterMarkus I suggest we try not to merge in changes to the config file that are not additions or refactoring of existing parameters, so that we keep it fixed. E.g. the new "Backhaul" property of the config is perfect, but perhaps you could revert the changes made to the other parameters that are not important to maintain track of.